src/ch/epfl/leb/sass/simulator/generators/realtime/Emitter.java

```diff
@@ -79,8 +87,10 @@
     protected PSF psf;
     
     /**
-     * The emitter's z-position.
+     * The emitter's position.

```

Comment:
Since I can safely remove the x and y attributes (that are duplicates of super attributes, as you pointed out), I'll just revert back this change completely.